lumicks/pylake/nb_widgets/kymotracker_widgets.py

```diff
@@ -732,57 +733,102 @@ def _get_default_parameters(kymo, channel):
+            "Positional search range",
+            "How much does the track fluctuate?",
             "float",
-            4 * position_scale,
-            *(0.0, 15.0 * position_scale),
+            2 * position_scale,
+            *(1.0 * position_scale, 5.0 * position_scale),
             True,
+            r"Search range defines how much the position of a particle is allowed to fluctuate "
+            r"from one time point to the next while still being considered part of the same "
+            r"track. Larger values will result in a wider range in which points are added to a "
+            r"track.",
         ),
         "window": KymotrackerParameter(
-            "Window", "How many frames can a track disappear.", "int", 4, *(1, 15), True
+            "Maximum gap",
+            "How many frames can a track disappear.",
+            "int",
+            4,
+            *(1, 15),
+            True,
+            r"Maximum gap defines how many scan lines without detected points can occur between "
+            r"two points and still be connected within a single track. This can occur, for "
+            r"instance, due to fluorophore blinking or the signal falling slightly below the "
+            r"intensity threshold. This value should be chosen such that small gaps in a track "
+            r"can be overcome and tracked as one, but not so large that separate tracks are strung "
+            r"together.",
         ),
-        "sigma": KymotrackerParameter(
-            "Sigma",
-            "How much does the track fluctuate?",
+        "min_length": KymotrackerParameter(
+            "Minimum length",
+            "Minimum number of frames a spot has to be detected in to be considered.",
+            "int",
+            3,
+            *(1, 10),
+            True,
+            r"Minimum length defines the minimum number of points a tracked line must contain for "
+            r"it to be considered valid. Reducing this parameter can be effective in reducing "
+            r"tracking noise. Note that this length refers to the number of detected points, not "
+            r"length in time!",
+        ),
+        "track_width": KymotrackerParameter(
+            "Expected spot size",
+            "How big a particle needs to appear to be tracked as a single molecule.",
             "float",
-            2 * position_scale,
-            *(1.0 * position_scale, 5.0 * position_scale),
+            4 * position_scale,
+            *(0.0, 15.0 * position_scale),
             True,
+            r"Expected spot size defines how big a bound particle needs to appear as on the "
+            r"kymograph for it to be tracked as a single molecule. This parameter should be set to "
+            r"roughly the width of the point spread function. Setting it larger rejects more "
+            r"noise, but at the cost of potentially merging tracks that are close together.",
         ),
         "vel": KymotrackerParameter(
```

Comment:
rename `vel` to `velocity`? it stands out as the only parameter key/name that is abbreviated.

Reply:
That would be a breaking change. Not one I want to piggyback on this PR.